### PR TITLE
[KOGITO-1728] - Release Kogito Operator/CLI 0.9.1

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>
See: https://issues.redhat.com/browse/KOGITO-1728
Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [X] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [X] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [X] Pull Request contains link to the JIRA issue
- [X] Pull Request contains description of the issue
- [X] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [X] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by radtriste** [comment]
community-operator PR has been merged: https://github.com/operator-framework/community-operators/pull/1592

File: pkg/apis/app/v1alpha1/zz_generated.openapi.go [hunk 1]
```diff
@@ -504,8 +504,10 @@ func schema_pkg_apis_app_v1alpha1_KogitoAppBuildObject(ref common.ReferenceCallb
 					"webhooks": {
 						VendorExtensible: spec.VendorExtensible{
 							Extensions: spec.Extensions{
-								"x-kubernetes-list-map-keys": "type",
-								"x-kubernetes-list-type":     "map",
+								"x-kubernetes-list-map-keys": []interface{}{
```
Comment (ricardozanini) [inline pkg/apis/app/v1alpha1/zz_generated.openapi.go:507]: This file shouldn't be updated.

**Comment by ricardozanini** [comment]
> community-operator PR has been merged: [operator-framework/community-operators#1592](https://github.com/operator-framework/community-operators/pull/1592)

Ideally, this PR should only be sent once everyone here approved this PR. But I understand we are on a special case, just a friendly reminder for the next time or when we have automation in place.

File: pkg/apis/app/v1alpha1/zz_generated.openapi.go (hunk 1 above)
Comment (spolti) [reply pkg/apis/app/v1alpha1/zz_generated.openapi.go:507]: I didn't upate it. it was auto generated.

**Comment by spolti** [comment]
> @spolti only the generated file that should not be updated. Also, have you reviewed the [CSV file](https://github.com/kiegroup/kogito-cloud-operator/blob/0.9.x/deploy/olm-catalog/kogito-operator/0.9.1/kogito-operator.v0.9.1.clusterserviceversion.yaml)? I've updated it in the 0.9.1 bump, but just to make sure! xD

I didn't find anything to update there, everything is pointing to 0.9.1, should I update anything else?

File: pkg/apis/app/v1alpha1/zz_generated.openapi.go (hunk 1 above)
Comment (ricardozanini) [reply pkg/apis/app/v1alpha1/zz_generated.openapi.go:507]: Something happens in your env/dependencies:
https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/apis/app/v1alpha1/zz_generated.openapi.go#L504-L509

**Pull request merged**